manifests/j/JetBrains/Hub/22.1.14638/JetBrains.Hub.locale.en-US.yaml

```diff
@@ -0,0 +1,33 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: JetBrains.Hub
+PackageVersion: 22.1.14638
+PackageLocale: en-US
+Publisher: JetBrains
+PublisherUrl: https://www.jetbrains.com
+PublisherSupportUrl: https://www.jetbrains.com/help/hub/Introduction-to-HUB.html
+PrivacyUrl: https://www.jetbrains.com/privacy-security
+Author: JetBrains s.r.o.
+PackageName: JetBrains Hub
+PackageUrl: https://www.jetbrains.com/hub
+License: https://www.jetbrains.com/hub/
```

Comment:
License doesn't seem quite right

Reply:
What should I keep? Proprietary?

https://www.jetbrains.com/help/hub/third-party-software.html#Hub_Front-End_Libraries